File: KantechGatewayApp/Jobs/NewProfileProcessor.vb
Namespace KantechGatewayApp.Jobs
    Public Class NewProfileProcessor
        Inherits BaseProcessor
        Protected Overrides Sub HandleRow(row As Dictionary(Of String, String))
            Infrastructure.Logger.Info($"[NewProfile] emp={row.GetValueOrDefault("employee ID")} name={row.GetValueOrDefault("name")} -> profile created", _logSub)
        End Sub
    End Class
End Namespace

File: KantechGatewayApp/Jobs/NewProfileProcessor.cs
using System.Collections.Generic;

namespace KantechGatewayApp.KantechGatewayApp.Jobs
{
    public class NewProfileProcessor : BaseProcessor
    {
        protected override void HandleRow(Dictionary<string, string> row)
        {
            Infrastructure.Logger.Info($"[NewProfile] emp={row.GetValueOrDefault("employee ID")} name={row.GetValueOrDefault("name")} -> profile created", _logSub);
        }
    }
}